diff --git a/mobile_main.py b/mobile_main.py
index 82602f6..d113783 100644
--- a/mobile_main.py
+++ b/mobile_main.py
@@ -24,16 +24,16 @@ from models.mobile_api_messages import BaseResponse, FavoriteCollection, Favorit
 from models.mobile_client import MobileClient
 from models.suggestion import Suggestion
 
-client_id_sitevar = Sitevar.get_by_id('appengine.webClientId')
-if client_id_sitevar is None:
-    raise Exception("Sitevar appengine.webClientId is undefined. Can't process incoming requests")
-WEB_CLIENT_ID = str(client_id_sitevar.values_json)
+# client_id_sitevar = Sitevar.get_by_id('appengine.webClientId')
+# if client_id_sitevar is None:
+#    raise Exception("Sitevar appengine.webClientId is undefined. Can't process incoming requests")
+WEB_CLIENT_ID = "{web-client-id}" # str(client_id_sitevar.values_json)
 ANDROID_AUDIENCE = WEB_CLIENT_ID
 
-android_id_sitevar = Sitevar.get_by_id('android.clientId')
-if android_id_sitevar is None:
-    raise Exception("Sitevar android.clientId is undefined. Can't process incoming requests")
-ANDROID_CLIENT_ID = str(android_id_sitevar.values_json)
+# android_id_sitevar = Sitevar.get_by_id('android.clientId')
+#if android_id_sitevar is None:
+#    raise Exception("Sitevar android.clientId is undefined. Can't process incoming requests")
+ANDROID_CLIENT_ID = "{android-client-id}" # str(android_id_sitevar.values_json)
 
 # To enable iOS access to the API, add another variable for the iOS client ID
 
