diff --git a/libTba/src/main/java/com/thebluealliance/api/call/TbaApiV2.java b/libTba/src/main/java/com/thebluealliance/api/call/TbaApiV2.java
index adaf158..6529c46 100644
--- a/libTba/src/main/java/com/thebluealliance/api/call/TbaApiV2.java
+++ b/libTba/src/main/java/com/thebluealliance/api/call/TbaApiV2.java
@@ -21,7 +21,7 @@ import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
-public interface DefaultApi {
+public interface TbaApiV2 {
   /**
    * District Events Request
    * Fetch a list of events within a given district
diff --git a/libTba/src/main/java/com/thebluealliance/api/rx/TbaApiV2.java b/libTba/src/main/java/com/thebluealliance/api/rx/TbaApiV2.java
index 1c73693..55d87a7 100644
--- a/libTba/src/main/java/com/thebluealliance/api/rx/TbaApiV2.java
+++ b/libTba/src/main/java/com/thebluealliance/api/rx/TbaApiV2.java
@@ -21,7 +21,7 @@ import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
-public interface DefaultApi {
+public interface TbaApiV2 {
   /**
    * District Events Request
    * Fetch a list of events within a given district
